src/crate/client/__init__.py

```diff
@@ -24,7 +24,7 @@
 
 # version string read from setup.py using a regex. Take care not to break the
 # regex!
-__version__ = "0.19.5"
+__version__ = "0.20.0"
```

Comment:
Yep I guess that would make sense.

We could also consider to use https://pypi.python.org/pypi/setuptools_scm